### DISCUSSION TITLE
Why is my taskbar transparent after updating?

### DISCUSSION THREAD

**Discussion by Kingternett** [valinet/ExplorerPatcher#1807] [Q&A] [answered]
I just updated my Explorer Patcher after receiving a Windows 11 update. Now my taskbar is totally transparent. Please tell me how to make my taskbar display a solid color instead

**Comment by pyrates999** [comment]
Did you install any applications from the windows app store that is for changing the transparency of the taskbar?

**Reply by Kingternett** [reply]
No. I did not install any apps for that purpose.

**Reply by pyrates999** [reply]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Reply by spetersen-ph** [reply]
I am having this same issue. I followed the instructions here and the issue is still persisting. 

Edit: I think it may be an interaction with classicshell. In classicshell I was able to enable taskbar styling and restore the color. The option was disabled though and I haven't used it before. Something about the new windows update must have changed something in the registry, not sure though really.

**Reply by pyrates999** [reply, marked as answer]
Uninstall Classic Shell.  That is abandonware.  It was replaced with Open Shell.

**Reply by MisaelK** [reply]
I was having the same issue, can confirm it was caused by Classic Shell. Didn't realize it has been abandoned. Open Shell seems to be working fine with Explorer Patcher. Thanks!

**Reply by Fivelssp** [reply]
Me too

**Reply by impeeza** [reply]
Bumping on a very old discussion, I just updated to EP 22621.3007.68.3 running Windows 11 22621.3296.  I did never downloaded nor used Classic Shell but my taskbar is transparent after updating just now to latest windows patch. 

Any toughs?  thanks.

**Reply by impeeza** [reply]
by the way despite the instructions, I just find which a Issue have the answer:

https://github.com/valinet/ExplorerPatcher/issues/2975

**Reply by DiablicaNati** [reply]
pls help me, how to HAVE transparent taskbar? I realy want to get this without any blur or anything

**Reply by Amrsatrio** [reply]
@NikiPl0708 I recommend installing TranslucentTB from the Microsoft Store.

**Comment by OsakaWebbie** [comment]
How can I determine whether I have Classic Shell installed, and if so, how do I uninstall it? This transparent Win10-style taskbar issue has now happened to me after a Windows update, and googling led me here. I don't remember installing Classic Shell (and had to even google to see what it was), but I did do some experimentation when I first got Win11 to try to restore my preferred taskbar style until I found EP (thanks for your software, Valinet!), so perhaps I've just forgotten.

**Reply by WilbaAtGitHub** [reply]
Go into Settings | Apps | Installed apps, and look for Classic Shell. I don't have it installed and have the same transparent taskbar problem.

**Reply by OsakaWebbie** [reply]
Thanks. I don't have it installed either, so now there are at least two of us for which there is apparently another cause of the transparent taskbar.

**Reply by callistosmoon** [reply]
Another in the same boat - no install of open or classic shell and a transparent taskbar.

**Reply by SthMax** [reply]
I am also encountering this problem after finishing Windows Cumulative Update KB5035853 - I do not install open shell or classic shell and get a transparent taskbar.

**Reply by Tularis1** [reply]
I have this issue too and can confirm it's nothing to do with Classic-Shell

**Reply by FreeAndNil** [reply]
Same here

**Comment by Vortilon7** [comment]
I have OpenShell and Explorer Patcher installed. Win 11 just applied this update: "2024-03 Cumulative Update for Windows 11 Version 23H2 for x64-based Systems (KB5035853)" and my taskbar has become transparent. I went into OpenShell and on the Taskbar tab, enabled "Customise Taskbar". I was then able to set the taskbar colour and now my Taskbar background is back. HTH.
![Taskbar Color setting](https://github.com/valinet/ExplorerPatcher/assets/163235202/8f6af802-be8d-4a3d-8cf2-905cdb2b3888)

**Reply by weeklyTea** [reply]
It solved my problem, thanks!

**Reply by krishna232** [reply]
it worked after installing open shell 

Thanks a ton

**Comment by marcatmadu** [comment]
here too after applying KB5035853 my taskbar (windows 10 style in explor. patcher) become transparent.  I do not have Open Shell installed. What now ?

Edit: I unistalled KB5035853 and everything is back to normal. So I guess this is a new issue to solve in E.P. ?

**Reply by DQuaN** [reply]
Confirmed KB5035853. Removing fixes it.

**Comment by serverside-is** [comment]
Me too. KB5035853 made part of the task bar invisible.

**Comment by stoeffu** [comment]
Same issue here after installing KB5035853. I hope it can be fixed witout removing KB5035853 by an explorer patcher fix.

**Comment by Amrsatrio** [comment]
For those experiencing transparent taskbar on the latest update, please install the latest prerelease.

**Reply by WonderDeer** [reply]
Thank you. I had the transparent taskbar issue after Win version 23H2 got installed. Downloading then installing the latest version of EP fixed it.

**Reply by gravitiq-cm** [reply]
Works for me too - thank you!

**Reply by serverside-is** [reply]
Worked like a charm. Thank you.

**Reply by LittlDog** [reply]
Thank you very much.

**Comment by marcatmadu** [comment]
> For those experiencing transparent taskbar on the latest update, please install the latest prerelease.

I cannot locate it.  I see a folder last mofified 10 hours ago but there's no compiled version in it.  Am I right? Do you have to compile the pre-release yourself?

**Reply by serverside-is** [reply]
https://github.com/valinet/ExplorerPatcher/releases

**Reply by marcatmadu** [reply]
thanks !   I noticed E.P. is now using my Windows Display Language!  I used to have it in English, now it's using Dutch. Any idea where I can change this ?  I looked all over every setting, didn't saw a language selection box or so.

**Reply by Fivelssp** [reply]
Tks, solved.

**Reply by impeeza** [reply]
how got solved?

**Reply by LittlDog** [reply]
Thank you very much^^ 
It Helped.

**Comment by WilbaAtGitHub** [comment]
I used Open-Shell to temporarily fix the transparency issue, but now can't get the start menu back to how it was before. Is there a global reset to undo what Open-Shell changed?

**Reply by impeeza** [reply]
as stated before there is an update, download latest release and upgrade manually

**Reply by WilbaAtGitHub** [reply]
That doesn't undo the changes I had to make with Open-Shell, for instance, the Start Menu Style only offers Classic, Classic with two columns, and Windows 7 options. I can't get it back to Windows 11 style.

**Reply by OsakaWebbie** [reply]
You can't uninstall Open Shell? Or you were already using it for other purposes and only want to undo this particular setting? If you were using it for other things but those things are easy to set up again, you might consider uninstalling and then starting over with a fresh install. (Sorry, I have never used it, so I can't help with specifics - I'm just brainstorming.)

**Reply by WilbaAtGitHub** [reply]
Yeah, sure. It seems to just change settings rather than dynamically interfering with how Windows operates, so uninstalling it would just remove the ability to change those settings in that way. The problem is, there doesn't seem to be a way to get back to the settings I had before I used it. I'm not using it for anything else.

**Reply by WilbaAtGitHub** [reply]
Uninstalling Open-Shell did the trick 😉  Now I just have to get my colours back...
